File: Fibonacci.cpp
#include<bits/stdc++.h>
using namespace std;

// int fib(int n){
//     if(n==1){
//         return 0;
//     } else if(n==2){
//         return 1;
//     } else {
//         return fib(n-1)+fib(n-2);
//     }
// }

// int main(){
//     int n;
//     cin>>n;
//     cout<<fib(n);
// }

int main(){
    int n;
    cin>>n;
    long long int a[n]= {0,1};
    for(int i=2; i<n;i++){
        a[i]=a[i-1]+a[i-2];
    }
    cout<<a[n-1];
}

File: Fibonacci.py
# n = int(input())


# def fib(k):
#     if k == 1:
#         return 0
#     elif k == 2:
#         return 1
#     elif k > 2:
#         return fib(k - 1) + fib(k - 2)


# print(fib(n))

n = int(input())

a = [0,1]

for i in range(2,n):
    a.append(a[i-1]+a[i-2])

print(a[n-1])
